eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -175,6 +175,13 @@ jobs:
           jdkVersionOption: $(JavaBuildVersion)
           jdkArchitectureOption: 'x64'
           publishJUnitResults: false
+      
+      - ${{ each artifact in parameters.Artifacts }}:
+        - template: /eng/common/pipelines/templates/steps/verify-changelog.yml
```

Comment:
This has the potential to break any libraries that don't currently have changelogs. @JimSuplizio Do we know if we have any such libraries any more?

Reply:
@JimSuplizio Where do you suggest I put it?